### PR TITLE
Migrate more JPA Query FAT to OL

### PR DESCRIPTION
**Pull request by jgrassel**
Migrated the rest of the LoopQuery (Ano) JPA Query FAT to Open Liberty.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#run-libby-bot
#build

**Review by dazey3 — CHANGES_REQUESTED** [review]
Couple minor things to note

File: dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/src/com/ibm/ws/query/web/loopqueryano/JULoopQueryAnoTest_001_Servlet.java [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+    }
+
+    @Test
+    public void jpa_spec10_query_svlquery_juloopquery_ano_test001_AMJTA_Web() throws Exception {
+        final String testName = "jpa_spec10_query_svlquery_juloopquery_ano_test001_AMJTA_Web";
+        final String testMethod = "testLoop001";
+        final String testResource = "test-jpa-resource-amjta";
+        executeTest(testName, testMethod, testResource);
+    }
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/src/com/ibm/ws/query/web/loopqueryano/JULoopQueryAnoTest_001_Servlet.java:66]: I thought about that, but decided against it.  This is testing how a JPQL is translated into SQL and the results generated.  It should be the same regardless whether it's container managed or resource-local.  Considering how long JPA tests are running, I felt that only going with AM-JTA is sufficient.

File: dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/src/com/ibm/ws/query/web/loopqueryano/JULoopQueryAnoTest_001_Servlet.java [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+    // Application Managed JTA
+    @PersistenceUnit(unitName = "QUERY_JTA")
+    private EntityManagerFactory amjtaEmf;
+
+    // Application Managed Resource-Local
+    @PersistenceUnit(unitName = "QUERY_RL")
+    private EntityManagerFactory amrlEmf;
+
+    private SetupQueryTestCase setup = null;
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/src/com/ibm/ws/query/web/loopqueryano/JULoopQueryAnoTest_001_Servlet.java:45]: Sure.

[the remark this answers is not in the document]
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/src/com/ibm/ws/query/web/loopqueryano/JULoopQueryAnoTest_001_Servlet.java:35]: Removed AM-RL and the CM-TS injections.

**Review by dazey3 — APPROVED** [review]
Looks fine

**Comment by jgrassel** [comment]
#run-libby-bot

**Pull request merged**